woocommerce-gateway-stripe.php

```diff
@@ -390,3 +390,34 @@ function woocommerce_gateway_stripe_init() {
 
 	woocommerce_gateway_stripe();
 }
+
+// Hook in Blocks integration. This action is called in a callback on plugins loaded, so current Stripe plugin class
+// implementation is too late.
+add_action( 'woocommerce_blocks_loaded', 'woocommerce_blocks_support' );
+
+function woocommerce_blocks_support() {
```

Comment:
This function should probably be namespaced a bit better. It's possible other plugins might think up this name for adding blocks support. So maybe something like `wc_stripe_woocommerce_blocks_support`?

Reply:
I know it's a bit long, but if we want to properly namespace the function I think we should go with the format `<gateway_name>_<function_name>`, i.e. `woocommerce_gateway_stripe_woocommerce_block_support`.